react/components/molecules/tabs-text/tabs-text.js

```diff
@@ -45,13 +46,37 @@ export class TabsText extends PureComponent {
             animatedBarOffset: undefined
         };
         this.tabLayouts = {};
+        this.XScroll = 0;

```

Comment:
There's only a single scroll so

```suggestion
        this.scroll = 0;
```